MetaMask/metamask-mobile
# chore: add privacy query params to Portfolio navigation

## **Description**

Added `metricsEnabled` and `marketingEnabled` query params to reflect user privacy preferences on navigation to Portfolio. These settings, sent from mobile, will be consumed by the Portfolio to update and align privacy preferences.

Our current privacy opt-in rate has been low since some recent updates to our privacy flow, so we would like to align our privacy settings with the other platforms.

## **Related issues**
Fixes:

## **Manual testing steps**

1. Go to Tokens page in mm mobile
2. Navigate to Portfolio (in a new tab, not in one that has already been opened)
3. See that the privacy params accurately reflect the user's settings in mobile

Example URL:

`https://portfolio.metamask.io/?metamaskEntry=mobile&metricsEnabled=false&marketingEnabled=false`

## **Screenshots/Recordings**



### **Before**


https://www.loom.com/share/d4034b05e4f7496d9f2ee55416777363

### **After**


With query params attached:

https://www.loom.com/share/4b382b19dcfe460a854ca4f95ebf4819

## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
